### ISSUE TITLE
Undo in issue key field

### ISSUE THREAD

**Issue by slarti-b** [carstengehling/jirastopwatch#74]
If you paste something in to the issue key field, it updates the URL to the issue key.  This is a really useful feature (IMO - and not juts because I proposed it! :) )

However, the magic re-write seems to kill the undo (Ctrl-Z) functionality.  I've had a few occasions now where I pasted into the wrong line (I think in combination with #59 since the dropdown button must be a sub-conscious visual target) and then had to try and remember what issue I had there...

Is it possible for the "extract issue key from URL" magic to work without breaking Ctrl-Z?  I will try and take a look myself if I get the time (either at this or #59 or both) but thought I'd open an issue in case someone else knows a quick fix.

**Comment by carstengehling** [comment]
Hi @slarti-b sorry for not responding before now. My life has been a bit chaotic for a few months, so unfortunately I've had very little energy to direct at StopWatch. :-(

I've noticed this behaviour as well - really strange... I'll see if I can figure out, why this happens. It bothers me as well.

**Comment by carstengehling** [comment]
I've found out, that the same behaviour goes for the simple TextBox when setting the .Text attribute. For TextBox you have the alternative of calling the Paste() method with the new text as parameter. This preserves the undo history. Unfortunately this method is not available for ComboBoxes. :-( I am very much thinking about developing my own custom ComboBox, since this will also enable me to do stuff like search/autocomplete...

**Comment by slarti-b** [comment]
Yeah, it would also allow a better solution to the row selection problem in #59 too :)
